src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,69 @@
-export const Tabs = () => {};
+import { FC } from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/tab';
+
+type PropsTabs = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: FC<PropsTabs> = (props) => {
+  const {
+    tabs,
+    selectedTabId,
+    onTabSelected,
+  } = props;
+
+  const isValidCurrentTabId = () => {
+    return tabs
+      .map(tab => tab.id)
+      .includes(selectedTabId);
+  };
+
+  const currentTabId = isValidCurrentTabId()
+    ? selectedTabId
+    : tabs[0].id;
```

Comment:
You can use [.find()](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/find) instead
Would be better to find selected tab here to avoid use 1 more find in `TabContent`.

```suggestion
  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```